### ISSUE TITLE
Snap maintenance: Release and dependency updates

### ISSUE THREAD

**Issue by dreua** [pdfarranger/pdfarranger#1121]
@soumyaDghosh who is responsible for keeping the dependencies/libraries of the snap build up to date? What are the best practices in the snapcraft ecosystem regarding this topic?
I don't like PDF Arranger being shipped with outdated libraries, there could be annoying bugs in the best case and security issues in the worst case.

https://github.com/pdfarranger/pdfarranger/blob/5e4db7cddccecdfe034841512cb9860892473921/snapcraft.yaml#L38-L50

**Comment by soumyaDghosh** [comment]
You can manually update them. I can't find in the repo itself, which versions are used now. Also, note that the repo, from where the snap will be published to the store is this one

https://github.com/pdfarranger/pdfarranger-snap

This one in this repo, is mainly for testing purposes.

**Comment by dreua** [comment]
No, I can't take on this task, first I'm already maintaining the Flatpak and Fedora packages, second I don't use snap myself.
For the flatpak I always use the latest versions, we don't specify exact version requirements because the latest versions are the best to use imo.

**Comment by AndyT-PrimoSpade** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by dreua** [comment]
> On a side note, from the statistics I’ve observed, the Snap package currently has the largest user base for pdfarranger on Linux.

Then it is even more important and urgent to keep the snap package up to date and in perfect shape, isn't it? 

>With that in mind, I’d like to humbly suggest that the upstream project consider adopting Snap as one of the official distribution formats, if you find it appropriate and up to standard.

What exactly do you mean by that? We already display the Snap option very prominently in the project readme, I don't think there is much more we can do.
